builtins/safe-default-configuration.json

```diff
@@ -728,6 +728,416 @@
       "name": "semantics",
       "namespace": "http://www.w3.org/1998/Math/MathML",
       "attributes": []
+    },
+    {
+      "name": "svg",
+      "namespace": "http://www.w3.org/2000/svg",
+      "attributes": [
+        {
+          "name": "viewBox",
+          "namespace": null
+        },
+        {
+          "name": "preserveAspectRatio",
+          "namespace": null
+        },
+        {
+          "name": "height",
+          "namespace": null
+        },
+        {
+          "name": "width",
+          "namespace": null
+        },
+        {
+          "name": "x",
+          "namespace": null
+        },
+        {
+          "name": "y",
+          "namespace": null
+        }
+      ]
+    },
+    {
+      "name": "g",
+      "namespace": "http://www.w3.org/2000/svg",
+      "attributes": []
+    },
+    {
+      "name": "defs",
+      "namespace": "http://www.w3.org/2000/svg",
+      "attributes": []
+    },
+    {
+      "name": "symbol",
+      "namespace": "http://www.w3.org/2000/svg",
+      "attributes": [
+        {
+          "name": "preserveAspectRatio",
+          "namespace": null
+        },
+        {
+          "name": "viewBox",
+          "namespace": null
+        },
+        {
+          "name": "refX",
+          "namespace": null
+        },
+        {
+          "name": "refY",
+          "namespace": null
+        },
+        {
+          "name": "height",
+          "namespace": null
+        },
+        {
+          "name": "width",
+          "namespace": null
+        },
+        {
+          "name": "x",
+          "namespace": null
+        },
+        {
+          "name": "y",
+          "namespace": null
+        }
+      ]
+    },
+    {
+      "name": "switch",
```

Comment:
I'm wondering if we should leave this out for now, given its capability to allow displaying one type of content to user A and another to user B.

Reply:
Good catch. I agree.

Reply:
Done.